packages/flutter/lib/src/material/divider.dart

```diff
@@ -87,23 +92,26 @@ class Divider extends StatelessWidget {
   /// ```
   static BorderSide createBorderSide(BuildContext context, { Color color, double width = 0.0 }) {
     assert(width != null);
-    return new BorderSide(
+    return BorderSide(
       color: color ?? Theme.of(context).dividerColor,
       width: width,
     );
   }
 
   @override
   Widget build(BuildContext context) {
-    return new SizedBox(
-      height: height,
-      child: new Center(
-        child: new Container(
-          height: 0.0,
-          margin: new EdgeInsetsDirectional.only(start: indent),
-          decoration: new BoxDecoration(
-            border: new Border(
-              bottom: createBorderSide(context, color: color),
+    return RotatedBox(
+      quarterTurns: isVertical ? 1 : 0,
+      child: SizedBox(
+        height: height,
+        child: Center(
+          child: Container(
+            height: 0.0,
+            margin: EdgeInsetsDirectional.only(start: indent),
+            decoration: BoxDecoration(
+              border: Border(
+                bottom: createBorderSide(context, color: color),
+              ),
             ),
           ),
         ),
```

Comment:
nit: please continue to leave the "new" keyword in there for now; we're going to remove them all at once very soon but in the meantime we want the codebase consistent.